client/post-editor/media-modal/test/index.jsx

```diff
@@ -44,6 +44,11 @@ jest.mock( 'calypso/my-sites/media-library', () =>
 	require( 'calypso/components/empty-component' )
 );
 
+const mockV4 = jest.fn();
+jest.mock( 'uuid', () => ( {
+	v4: () => mockV4(),
```

Comment:
Shouldn't this be inline? I don't believe `jest.mock` implementations are allowed to reference functions in the larger scope (Jest should throw a warning or error about this I believe, unless it's changed).

I think the correct way to do this is to just import `uuid` in the test and it will be mocked.

Reply:
It seems to be working just fine 😉  And we need it like this to be able to mock different calls in specific tests [like we do here](https://github.com/Automattic/wp-calypso/pull/52672/files#diff-5afaede1c46e80dfcf7ce7733960c7c91d36b16811eea183a7ddf7401b7e994fR353).

Reply:
> Edit: never mind, this appears to no longer be the case!

👍